Assets/Scripts/UI/LaborOrderPanelManager.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+public class LaborOrderPanelManager : MonoBehaviour
+{
+
+    [SerializeField, HideInInspector]
+    public GameObject buttonContainer;
+
+    [SerializeField]
+    GameObject button_prefab;
+
+    public GameObject[,] buttonList;
+    private int buttonTotal;
+    private bool completed = false;
+
+    void Start()
+    {
+
+        // Adjust grid layout of buttons
+        buttonContainer = GameObject.Find("Buttons");
```

Comment:
This is a bit of a tenuous link. In the future, it may be better to just drag and drop the editor GameObject into this field.

Reply:
Alright, I will make sure to make unit tests and code coverage.

As for the LaborOrderPanelManager, do you know of a way to make sure a script starts after one script is finished completing? The reason that I did that was to make sure that another script, LaborOrderManager_VM, finishes one of their functions before my script, LaborOrderPanelManager, starts.

Lastly, I will keep in mind those tenuous links in the future.